ui/nuxeo-routing-behavior.js

```diff
@@ -71,22 +71,29 @@ export const RoutingBehavior = {
   _computeUrlFor() {
     return function(...args) {
       if (this.router) {
-        const route = args[0];
         const baseUrl = this.router.baseUrl || '';
+        let [route, ...params] = args;
+        if (!route) {
+          return '';
+        }
+        if (typeof route === 'object') {
+          if (!this.router.resolveRoute) {
+            throw new Error('cannot resolve object: no route resolver is available');
+          }
+          const additionalParams = params;
+          ({ route, params } = this.router.resolveRoute(route));
```

Comment:
What's the use of the outer `()` here?

Reply:
Not valid syntax without the `()`. Maybe due to the [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)?

Reply:
Exactly that! Lint won't allow it.